### DISCUSSION TITLE
Deletion webhooks does not include the payload

### DISCUSSION THREAD

**Discussion by medteck** [directus/directus#6027] [Feature Requests]
When you create a [Delete] webhook for a collection, and then you actually delete one element of this collection, the webhook is actually fired but is useless since it only includes the id of the deleted element. This id is no longer relevant because it does not help retrieve any information about the deleted data (because it's deleted).

The webhook should include at least the payload of the deleted record.

**Steps to reproduce:**
- Create a webhook for any collection and select "Delete" as an action.
- Use [https://webhook.site](https://webhook.site) to generate an endpoint, then set the generated endpoint for the webhook
- Delete one record of the collection you've set the webhook for
- Go back to webhook.site and reload the page to see what directus have sent throught the webhook event.

**Environment**
Version: Directus 9.0.0-rc.69
OS: Ubuntu Server 20.04

**Comment by hnnazm** [comment]
Any workaround for this? Can’t find any resources to solve this issue.

**Reply by rijkvanzanten** [reply]
Use flows instead of webhooks 👍🏻 You can read the item payload in the filter event before its deleted and then send the payload elsewhere with the `request` operation

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
